src/statements/CassandraStatements.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+/** Create an INSERT prepared statement object.
+ * 
+ * Preparing queries gets the best performance and will ensure that
+ * the JavaScript parameters are correctly mapped to Cassandra types.
+ * 
+ * http://docs.datastax.com/en/developer/nodejs-driver/3.2/features/batch/
+ * This will be used for the batch() method, so the object needs to have
+ * two properties: query and params
+ * 
+ * @param {{keyword: string, lang_code: string, ?translations: map<string,string>}}
+ * @returns {{mutation: string, params: Array<string|map<string,string>>}}
+ */
+function prepareInsertTopic(topic) {
+  if(!topic.keyword || !topic.lang_code) return null;
```

Comment:
General remark: you also may want to check for nullity of the wrapper object itself: `if (!topic || !topic.keyword || !topic.lang_code) return null;`

As a more general comment: note that the `!variable` syntax is a bit of a dangerous habit since it'll break for number values so often we prefer `variable != null` which checks just for `null` and `undefined`. Here, using `!variable` is fine though since you also want to disallow empty strings.